spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/Container.scala

```diff
@@ -16,11 +16,12 @@
 
 package com.intel.analytics.bigdl.nn
 
-import com.intel.analytics.bigdl.Module
+import com.intel.analytics.bigdl.nn.Graph.ModuleNode
```

Comment:
we should not import Graph here, right?

Reply:
just want to use the shortcut definition of `ModuleNode`. somehow it can be removed.